docker/stellargraph-demos/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.6-slim as base
```

Comment:
Why this as a base?

Reply:
I'm using a multi-stage build—so the first intermediate image install the packages needed to download the files and the 2nd actual image (which the the `stellargraph/stellargraph` image just built) just copies the files from the first

Reply:
Right, but why python:3.6-slim? This stage doesn't seem to use any python?

Reply:
Ah I see—I guess it's to keep it consistent. The image would already live in the agent from the `stellargraph/stellargraph` build—and it's going to be used for the actual build, so it makes sense to use it. What are your thoughts? We can update it to use `alpine`?

I also want to update the images used in the BK pipeline from `python3.6` to the slim version in a different PR

Reply:
Ah makes a lot of sense. Maybe a `# image needed anyway, so also use for downloading` comment or something?

Reply:
Yeah sure, I'll add a comment 👍